### DISCUSSION TITLE
How do you edit a wireguard connection

### DISCUSSION THREAD

**Discussion by Gooberpatrol66** [celzero/rethink-app#1468] [Q&A] [answered]
After you create a connection, there's only a button to enable/disable it, not to edit the settings

**Comment by Gooberpatrol66** [comment]
Also no apparent way to delete a connection

**Comment by hussainmohd-a** [comment, marked as answer]
If you click on the WireGuard name, it will take you to the configuration screen where you can edit the configuration. There is also a delete button that allows you to remove the configuration.

**Reply by Gooberpatrol66** [reply]
Tapping on the name doesn't do anything
